### ISSUE TITLE
KE driver schemas shows all fields can be updated

### ISSUE THREAD

**Issue by dramich** [rancher/rancher#18803]
Master 3/11/19

Currently when looking at `v3/schemas/amazonElasticContainerServiceConfig` all `resourceFields` shows as `update: true`. This is not correct as each driver has it's own list of fields that are actually able to be updated and the schema should reflect this. 

UI work will also need to be completed in order to make the cluster edit screen dynamic so a user is not confused by being able to update a field yet no change ever occurs in the provider.

**Comment by cjellick** [comment]
I dont think we can/should tackle this in v2.3

**Comment by deniseschannon** [comment]
With KEv2 being introduced for the hosted providers, this would hopefully have been addressed. 

@rmweir Can you confirm if this was addressed when we refactoring EKS?

**Comment by rmweir** [comment]
@deniseschannon Yep, going forward the norman `noupdate` or `nocreate` tags just need to be added to the structs. This populates the schema correctly. Although, those tags cause input just to be dropped even if they are included in a request through curl or something instead of an error message I believe. Also, I think he UI for EKSv1 in 2.4 was changed to not show fields that cannot be updated as editable.

I have heard talk before about those tags going away once/if norman goes away so we'll have to figure out something if that does happen.
